apps/settings/test/unit/icc_menu_test.js

```diff
@@ -148,4 +149,35 @@ suite('STK (Main menu) >', function() {
         '#icc-entries li:nth-child(2) small').textContent, 'SIM 2');
     });
   });
+
+  suite('Remove STK Menu >', function() {
+    var emptyMenu = {
+      'title': 'DummyOperator',
+      'items': [],
+      'presentationType': 0
+    };
```

Comment:
I think it could be better to put this empty menu definition near the dummy menu in line 23 [1] since it could be used in other tests in the future. WDYT?

[1] https://github.com/mancas/gaia/blob/bug1117656/apps/settings/test/unit/icc_menu_test.js#L23